main.go

```diff
@@ -104,6 +105,7 @@ func init() {
 	flag.BoolVar(&force, "force", false, "force overwrite existing files")
 	flag.BoolVar(&force, "f", false, "force overwrite existing files")
 
+	flag.BoolVar(&help, "help", false, "print usage and exit")
```

Comment:
True, just I didn't manage to get it exit with retcode=0.  Anyway it seems to be the way flag works so I have just removed this part from the patch (and made it even more trivial). I've left the part that --version works when no other args are provided.  Thanks for the quick review!